src/RuleCommonITILObject.php

```diff
@@ -127,6 +129,29 @@ public function getTitleAction()
                 </div>
 TWIG, ['message' => __('Urgency or impact used in actions, think to add Priority: recompute action if needed.')]);
         }
+
+        // Validation step assignation (validation step or threshold) without validation creation
+        $action_keys = [];
+        $actions_fields = array_column($this->actions, 'fields');
+        foreach ($actions_fields as $field) {
+            $action_keys[] = $field['field'];
+        }
+
+        $fields_related_to_validation_but_not_triggering_validation = ['validationsteps_id', 'validationsteps_threshold'];
+        $fields_trigerring_validation = ['users_id_validate', 'responsible_id_validate', 'groups_id_validate', 'groups_id_validate_any', 'users_id_validate_requester_supervisor', 'users_id_validate_assign_supervisor'];
+        if (
+            array_intersect($fields_related_to_validation_but_not_triggering_validation, $action_keys)
+            && empty(array_intersect($action_keys, $fields_trigerring_validation))
+        ) {
+            // language=Twig
+            echo TemplateRenderer::getInstance()->renderFromStringTemplate(<<<TWIG
+                <div class="alert alert-warning">
```

Comment:
Should use alert Twig macros.

Reply:
which macro ?

Reply:
`templates/components/alerts_macros.html.twig` is the template and `alert_warning` is the macro. I will say though that if it isn't important enough to have the warning read aloud immediately after it is shown on the page by a screen reader, then it is OK as it is now. Without `role="alert"`, the alert will only be read aloud when the user navigates to it.